core/src/test/scala/reftree/diagram/AnimationSpec.scala

```diff
@@ -0,0 +1,32 @@
+package reftree.diagram
+
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.util.Random
+
+class AnimationSpec extends FlatSpec with Matchers {
+
+  "iterateUntilFixPoint" should "expand until fixpoint" in {
+
+    val seed = 1337
+    val maxTests = 1200
+
+    (0 until 4200).foreach { _ =>
```

Comment:
Ah of course, I didn't see you already had scalacheck as a dependency so I didn't want to add it :)